master/docker-entrypoint.sh

```diff
@@ -6,14 +6,9 @@ function start_dgraph() {
     dgraph server --memory_mb=2048 --zero=localhost:5080 &
 }
 
-function start_dgraph_web {
-    dgraph-ratel &
-}
-
 start_dgraph
 
-if [ "$1" = 'dev' ]; then
-    start_dgraph_web
-fi
+# Give dgraph some time to come up
+sleep 15
```

Comment:
I really don't like such delay constructs, because they try to circumvent shortcomings. 
So what is the actual problem and what is the base for the assumption  15?

Reply:
When i tried Giasemi's change the container died because qmstr-master was trying to connect to a not-ready Dgraph. If you start Dgraph it takes ~10 seconds to see the message "About to serve".